emails/tests/fixtures/disabled_mask_for_spam_expected.email

```diff
@@ -0,0 +1,289 @@
+Subject: =?utf-8?q?=E2=81=A8Firefox_Relay=E2=81=A9?= has disabled one of your
+ email masks.
+From: reply@relay.example.com
+To:
```

Comment:
🔴 The blank "To" looks like trouble. Could be test setup, could be a logic error.

Reply:
🔴 this is still blank

Reply:
Found and fixed the blank `To:` bug. It was a bug in the test. https://github.com/mozilla/fx-private-relay/pull/5045/commits/53855a3d12dcede9d3b026bdd42e2ec125c9766f